dev-utils/Cargo.toml

```diff
@@ -15,3 +19,7 @@ serde_json = "1.0"
 serde = { version = "1.0", features = ["derive"] }
 chrono = { version = "0.4", features = ["serde"] }
 kvdb-memorydb = "0.1"
+dirs = "2.0.1"
+read_input = "0.8"
+quest = "0.3"
```

Comment:
Aren't `quest` and `read_input` different crates for doing same thing? If yes, we can remove one of them.

Reply:
quest: for password
read_input: normal input
password input is not supported in read_input

Reply:
Normal input is also supported by quest. Can we only use quest here?

Reply:
agree, i will change it to quest